pandas/io/formats/printing.py

```diff
@@ -409,7 +438,44 @@ def best_len(values):
     return summary
 
 
-def format_object_attrs(obj):
+def _justify(head, tail):
+    """
+    Justify each item in each list-like in head and tail, so each item
+    right-aligns when the two list-likes are stacked vertically.
+
+    Parameters
+    ----------
+    head : list-like of list-likes of strings
```

Comment:
Format here should be `object : type` and the description should come on the next line (applicable to all items here)